contents/teams/surveys/objectives.mdx

```diff
@@ -1,37 +1,62 @@
+You asked — we listened. We're making survey analysis more useful and enjoyable, starting with a brand new global analytics page.
 
-- Reduce [SLA](https://en.wikipedia.org/wiki/Service-level_agreement) to fix bugs from months to weeks
-- Reduce [paper cut](https://en.wikipedia.org/wiki/Lingchi) issues by 50%
-    - Currently around ~30 [open bugs](https://github.com/PostHog/posthog/issues?q=is%3Aopen+is%3Aissue+label%3Afeature%2Fsurveys+label%3Abug)
+- [Global analytics view](https://github.com/PostHog/posthog/issues/28418)
+- A cleaner, more useful survey responses page
+- Extra polish and UX improvements across the board
 
-# Goal 3: Finish up [WIP](https://en.wikipedia.org/wiki/Work_in_process) features
+# 🎨 Goal 3: Survey customization
 
-We've started and nearly finished a couple of features our customers requested, but they are not [GA](https://en.wikipedia.org/wiki/Software_release_life_cycle#General_availability_(GA)) yet, the goal is to finish them up, gather feedback and put them in front of customers.
+A lot of people use the API to tweak their surveys. Let’s bring those use cases into the product.
 
-- [Adaptive Sampling](https://github.com/PostHog/posthog/issues/25722)
-    - Currently under a feature flag
-- [Partial Responses](https://github.com/PostHog/posthog/issues/19300)
-    - There are Draft PRs
-- [Survey Actions](https://github.com/PostHog/posthog/issues/27378)
-    - Currently under a feature flag
-- [AI Summaries](https://github.com/PostHog/posthog/issues/27379)
-    - Currently under early access features
+- Better wait period controls
+- [Alignment customization](https://github.com/PostHog/posthog/issues/28479)
+- [Inline surveys](https://github.com/PostHog/posthog/issues/30097)
+- A handful of other power-user options
 
-# Goal 4: Mobile Surveys
+# 💬 Goal 4: Make Key Customers Super Happy
 
-We want to bring the same functionality to Mobile that we have for Web Surveys, [GH Issue](https://github.com/PostHog/posthog/issues/25085).
+We’ve had some great convos with customers — now let’s turn those into lasting partnerships.
 
-- React Native
-- iOS
-- Android
-- Flutter
+- Build a case study with a power user
+- Be lightning-fast at resolving their issues
+
+# 🧰 Goal 5: Templates, Templates, Templates
+
+Right now, duplicating and customizing surveys across projects/environments isn’t great. Let’s fix that.
+
+- [Save custom templates](https://github.com/PostHog/posthog/issues/27309)
+- [Duplicate surveys across projects/environments](https://github.com/PostHog/posthog/issues/29138)
+- Add mobile-specific templates
+- Explore using AI to create survey templates
+
+# 🧑‍💻 Goal 6: Hire a Cracked Product Engineer
+
+The team’s growing! We’re hiring a Product Engineer to help us build faster and better.
+
+- Hire and onboard our next teammate
+- Make onboarding smooth and fast so they can hit the ground running
+
+# ✨ Goal 7: Improve the Onboarding UX
```

Comment:
Lucas and possible new hire, but really anyone from Surveys team can pick it up

Reply:
i'd move this to side quest, so we have 5 product goals + 1 hire